docs/testing-guide.md

```diff
@@ -0,0 +1,119 @@
+# Testing Built Images
+
+Images may be run on either bare metal or virtualization. The following guide provides examples to speed
+development and testing.
+
+[libvirt](https://libvirt.org/) is a commonly available virtualization toolkit. Much of the development of EIB
+has been tested against libvirt running on an [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/)
+installation. Other hypervisors may be used, but this guide will cover the specifics of libvirt in particular.
+
+## Testing Self-installing ISOs
+
+If you're already familiar with using libvirt to create images installed from ISOs, you can skip to the section
+about [resetting the VM between tests](#Resetting-the-VM-for-Self-install-ISO-Testing ).
+
+> :warning: libvirt may automatically remove the installation media after the first boot. Upon reboot, the installation
+> ISO, and more importantly its embedded combustion configuration, will not be present and combustion won't run. If
+> the JeOS installer begins, the first step in debugging should be to ensure the installation media is still present
+> after the initial installation writes the SLE Micro raw image to disk.
+
+Using Virtual Machine Manager, select the option to create a new VM. 
+
+Select the "Local install media (ISO image or CDROM)" option:
+
+![image](./images/libvirt-iso-1.png)
+
+Browse to the ISO built by EIB. If Virtual Machine Manager is unable to detect the operating system, type `slem5.5`
+to search for the "SUSE Linux Enterprise Micro 5.5" version:
+
+![images](./images/libvirt-iso-2.png)
+
+Enter the memory and CPU settings as appropriate for your host:
+
+![images](./images/libvirt-iso-3.png)
+
+Ensure "Enable storage for this virtual machine" is selected and enter an image size:
+
+![images](./images/libvirt-iso-4.png)
+
+Name the VM and select Finish:
+
+![images](./images/libvirt-iso-5.png)
+
+After pressing "Finish", the VM will boot. Depending on the version of the SLE Micro self-install ISO, it may
+prompt you to manually select the "Install SLE Micro" option before proceeding. In either case, stop the installation
+by using the "Shutdown -> Force Off" option under the "Virtual Machine" menu. See the next step for more information
+of what needs to happen before the installation can continue.
```

Comment:
I think "information of" should be "information on" or "information about"

Reply:
Yup, this should be "on". Good eye.